src/App.tsx

```diff
@@ -67,18 +67,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && isLoaded && !hasError && posts.length === 0 && (
                   <div className="notification is-warning" data-cy="NoPostsYet">
                     No posts yet
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length > 0 && (
-                  <PostsList
-                    posts={posts}
-                    selectedPostId={selectedPost?.id}
-                    onPostSelected={setSelectedPost}
-                  />
+                {author && isLoaded && !hasError && posts.length > 0 && (
```

Comment:
We can only display posts when the list is greater than 0.
Other checks are unnecessary, because this request only occurs when the author is selected.
If there are errors or an active loader - the `list === 0`
```suggestion
                {posts.length && (
```